src/main/kotlin/org/jitsi/jibri/CallUrlInfo.kt

```diff
@@ -53,6 +54,6 @@ data class CallUrlInfo(
 
     override fun hashCode(): Int {
         // Purposefully ignore urlParams here
-        return Objects.hash(baseUrl.toLowerCase(), callName.toLowerCase())
+        return Objects.hash(baseUrl.lowercase(Locale.getDefault()), callName.lowercase(Locale.getDefault()))
```

Comment:
did you explicitly pass the locale in for a reason? i see it supports a plain version without passing the locale as well.

Reply:
That's what IntelliJ's auto-fix of the deprecation warning did -- I can change it.